<launch>
    <rosparam command="load" file="$(find hummingbird_pkg)/config/hummingbird_ppo_params_baseline.yaml" />
    <!-- Launch the training system -->
    <node pkg="hummingbird_pkg" name="hummingbird_training_ppo_baseline" type="train_hummingbird_ppo_baseline.py" output="screen"/> -->
</launch>
